.github/workflows/ci.yml

```diff
@@ -5,15 +5,18 @@ name: build
 
 on:
   push:
-    branches:
-      - develop
+    branches: [develop, 'GEN*', 'gen*']
```

Comment:
Just a comment to note that `test` and `lint` will run for every push now to the feature branches regardless of whether there is a PR or not. Nothing to fix here.